src/blueteam/Robot.java

```diff
@@ -34,6 +34,9 @@ void run() throws GameActionException {
 			checkHealth();
 			dodge();
 			step();
+			if (getRobotCount(RobotType.ARCHON) == 0)
+				if (rc.getTeamBullets() > 900)
+					rc.donate(rc.getVictoryPointCost());
```

Comment:
I would move whole process of buying victory points to robot. I think that current archon implementation is currently OK.